week10/bulletin-board/src/pages/Detail.jsx

```diff
@@ -0,0 +1,352 @@
+import React, { useState, useEffect } from "react";
+import { LoadPostDetailApi, PostEditApi, DeletePostApi } from "../api/post";
+import { GetHeartCountApi, AddHeartApi, RemoveHeartApi } from "../api/heart";
+import {
+  AddCommentApi,
+  DeleteCommentApi,
+  LoadCommentsApi,
+} from "../api/comment";
+import { useParams, useNavigate } from "react-router-dom";
+import { MdDeleteOutline } from "react-icons/md";
+import styled from "styled-components";
+
```

Comment:
2. 개별 게시글을 렌더링하는 컴포넌트에서 JWT 테스트 API를 불러옵니다.

```suggestion
import { JwtTest } from "../api/user";
```